components/software_sensor_iso_supported.mdx

```diff
@@ -0,0 +1,79 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+This sensor is included in the Clearpath Robotics operating system that is installed on all our robots.
+This allows you to add or remove the sensor from your robot's software description by changing the environment variables in `/etc/ros/setup.bash`.
+
+#### Refer to your robot's README file for more details:
```

Comment:
As another minor nitpick, as there are several README files, I think we note that we are referring specifically to the robot _description_ README file.